lib/protocol/bep.proto

```diff
@@ -100,6 +100,7 @@ message FileInfo {
     uint32       permissions    = 4;
     int64        modified_s     = 5;
     int32        modified_ns    = 11;
+    string       modified_by    = 13;
```

Comment:
We want short id here for the sake of saving database space.
Short ID is essentially first 5 characters of long ID, so you can look up device name by matching short ID as prefix to each devices lond ID.